### DISCUSSION TITLE
Change time reminders are sent

### DISCUSSION THREAD

**Discussion by KierenConnell** [monicahq/monica#5842] [Q&A]
Is it possible to change the time reminders are sent out?

**Comment by tlwt** [comment]
Did you find out anything? I have the same question.

**Comment by tlwt** [comment]
@KierenConnell - under settings in the block "international settings", there is a field "Time of the day reminders will be sent".

![reminder](https://user-images.githubusercontent.com/8267086/212268311-0b017a54-f83b-4596-b188-586453d7c320.png)
